syntax = "proto3";

message Player {
  string id = 1;

  string name = 2;

  string team = 3;

  string position = 4;

  int32 salary = 5;

  float points = 6;

  string opposing_team = 7;

  float sim_points = 8;
}

message PlayerPool { repeated Player players = 1; }